task/dump-agent/src/udp.rs

```diff
@@ -124,16 +123,11 @@ impl ServerImpl {
                 meta,
                 &tx_data_buf[..meta.size as usize],
             ) {
-                // We'll drop packets if the outgoing queue is full or the
-                // server has died; the host is responsible for retrying.
-                //
-                // Other errors are unexpected and panic.
                 ringbuf_entry!(Trace::SendError(e));
                 match e {
+                    // We'll drop packets if the outgoing queue is full or the
+                    // server has died; the host is responsible for retrying.
```

Comment:
I'm not sure if we actually have a retry implementation in `humility`, but this change doesn't make things any worse.